views/db/organizations/departments/show.erb

```diff
@@ -0,0 +1,79 @@
+<% content_for :breadcrumbs  do %>
+  <nav class="breadcrumb" aria-label="breadcrumbs">
+    <ul>
+      <li><a href="/">Home</a></li>
+      <li><a href="/db">Database</a></li>
+      <li><a href="/db/organizations">Organization Department</a></li>
+    </ul>
```

Comment:
Should probably link back to the `/db/organizations/:id` in the bread crumbs.

Reply:
Maybe also include a link to `/db/organization_departments/:id` for the `Organization#parent_department`.